build/postgres-operator/Dockerfile

```diff
@@ -1,15 +0,0 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal
```

Comment:
Is there anything else in this dir to save?

Reply:
Oh, yeah -- GH was saying this file was deleted, but I see on your branch the whole dir is gone, which is what I was hoping would be the case. Just wanted to check.